src/analytics/private/sqls/redshift/user-v2.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS {{schema}}.user_v2 (
+    event_timestamp timestamp not null,
+    user_pseudo_id varchar(255) not null,
+    user_id varchar(255),
+    user_properties super,
```

Comment:
Do we also need a string column to store custom user properties?

Reply:
@luorobin-a2z do we need a string column e.g.  `user_propertie_json_str` in user table?

Reply:
Let's add it.

Reply:
added